addons/skin.estuary/xml/DialogVideoInfo.xml

```diff
@@ -259,23 +259,45 @@
 					</include>
 				</control>
 				<control type="group">
-					<visible>Integer.IsEqual(Container(50).NumItems,0) + String.IsEmpty(ListItem.Art(tvshow.banner)) + String.IsEmpty(ListItem.Art(banner))</visible>
-					<control type="image">
-						<left>0</left>
-						<top>164</top>
-						<width>1245</width>
-						<height>328</height>
-						<texture border="21">dialogs/dialog-bg.png</texture>
+					<control type="group">
+						<visible>!Container.Content(Sets) + Integer.IsEqual(Container(50).NumItems,0) + String.IsEmpty(ListItem.Art(tvshow.banner)) + String.IsEmpty(ListItem.Art(banner))</visible>
+						<control type="image">
+							<left>0</left>
+							<top>164</top>
+							<width>1245</width>
+							<height>328</height>
+							<texture border="21">dialogs/dialog-bg.png</texture>
+						</control>
+						<control type="textbox">
+							<left>17</left>
+							<top>185</top>
+							<width>1210</width>
+							<height>283</height>
+							<align>center</align>
+							<aligny>center</aligny>
+							<font>font36_title</font>
+							<label>$LOCALIZE[31079]</label>
+						</control>
 					</control>
-					<control type="textbox">
-						<left>17</left>
-						<top>185</top>
-						<width>1210</width>
-						<height>283</height>
-						<align>center</align>
-						<aligny>center</aligny>
-						<font>font36_title</font>
-						<label>$LOCALIZE[31079]</label>
+					<control type="group">
+						<visible>Container.Content(Sets) + Integer.IsEqual(Container(50).NumItems,0)</visible>
+						<control type="image">
+							<left>0</left>
+							<top>164</top>
+							<width>1245</width>
+							<height>328</height>
+							<texture border="21">dialogs/dialog-bg.png</texture>
+						</control>
+						<control type="textbox">
+							<left>17</left>
+							<top>185</top>
+							<width>1210</width>
+							<height>283</height>
+							<align>center</align>
+							<aligny>center</aligny>
+							<font>font36_title</font>
+							<label>$LOCALIZE(31168)</label>
+						</control>
```

Comment:
instead of duplicating a code block only because the label differs, define a variable for the label. 
`<label>$VAR[VideoInfoNoResult]</label>`
```
	<variable name="VideoInfoNoResult">
		<value condition="Container.Content(Sets)">$LOCALIZE[31168]</value>
		<value>$LOCALIZE[31079]</value>
	</variable>
```